exchange/utils.go

```diff
@@ -28,6 +30,18 @@ func cleanOpenRTBRequests(orig *openrtb.BidRequest, usersyncs IdFetcher, blables
 	}
 
 	requestsByBidder, errs = splitBidRequest(orig, impsByBidder, aliases, usersyncs, blables, labels)
+
+	// Clean PI from bidrequests if not allowed per GDPR
+	gdpr, consent, _ := extractGDPR(orig, usersyncIfAmbiguous)
+	// TODO : do something with the error value
```

Comment:
Can just delete the `TODO` if you're not going to fix it now... these never get done later.

IMO PBS should probably just act by `usersyncIfAmbiguous` on error though.

Reply:
actually... now that you mention it, I think we _do_ kill it completely.

- https://github.com/prebid/prebid-server/blob/master/endpoints/openrtb2/auction.go#L670
- https://github.com/prebid/prebid-server/blob/master/endpoints/openrtb2/auction.go#L641

So... doesn't really matter how you handle errors here. Input validation is the first thing we do.